AndroidManifest.xml

```diff
@@ -15,7 +15,7 @@
     <uses-feature android:name="android.hardware.camera.autofocus" android:required="false"/>
 
     <!-- Application Uses SDK -->
-    <uses-sdk android:minSdkVersion="9" android:targetSdkVersion="10"/>
+    <uses-sdk android:minSdkVersion="11" android:targetSdkVersion="10"/>
```

Comment:
What's the reason behind this upgrade from 9 to 11 ?
Target version should be upgraded too then.